### ISSUE TITLE
Indication of presence of Alt Text without having to scroll mouse over it

### ISSUE THREAD

**Issue by libertyforward1** [enafore/enafore#34]
Hello! In similar fashion to https://github.com/nolanlawson/pinafore/issues/2250, it would be lovely if there was a toggle to invoke some sort of border or indication of alt text being present with visual media.

To address a point Nolan brought up in that original ticket, it isn't my intent to "shame" anyone for using or not using alt text, but its presence _may_ make me more likely to boost a post. However I'm finding myself spending more and more time going back and forth from the keyboard to the mouse to keyboard to mouse and so on to determine whether or not alt text is present.

Having a vague "button" present indicating alt text is difficult for me to see personally, and I'm not visually impaired (with glasses anyway). Having a nice, red border to clearly indicate missing alt-text would save me an immense amount of time.

Thanks!
